CHANGELOG.md

```diff
@@ -75,7 +75,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed the deprecated `terminate_on_nan` argument from the `Trainer` constructor ([#12553](https://github.com/PyTorchLightning/pytorch-lightning/pull/12553))
 
 
--
+- Removed the deprecated `gpu_metrics` argument from the `LoggerConnector` constructor ([#12657](https://github.com/PyTorchLightning/pytorch-lightning/pull/12657))
```

Comment:
You missed removing `Trainer(log_gpu_memory)` which is what we want to mention here in the CHANGELOG.

Pushed the removal for you

Reply:
Thank you!